MetadataExtractor/Formats/QuickTime/QuickTimeReaderExtensions.cs

```diff
@@ -58,5 +58,21 @@ public static decimal Get32BitFixedPoint([NotNull] this SequentialReader reader)
                 reader.GetUInt16(),
                 decimal.Divide(reader.GetUInt16(), ushort.MaxValue));
         }
+
+        public static int[] GetMatrix(this SequentialReader reader)
```

Comment:
I think this should return `float[]`, and the below divisions be done with floats rather than ints to avoid losing precision. It might not matter for rotations of 0/90/180/270 degrees, but in general it seems truer to the fixed encoding.

Reply:
You are right, I've changed that by reading the 16.16 signed value more straight forward.